.github/workflows/generate-release-notes.yml

```
@@ -59,6 +59,41 @@ jobs:
         with:
           script: return "${{ inputs.version }}".replace(/^(\d+\.\d+).+$/m, "$1")
           result-encoding: string
+      - name: Make checklist body
+        id: make-checklist
+        uses: actions/github-script@v8
+        with:
+          script: |
+            if (/^8\./.test("${{ inputs.version }}")) {
+              if ("${{ github.repository }}" === "elastic/beats") {
+                return process.env.BEATS_8
+              } else {
+                return process.env.EA_FLEET_8
+              }
+            } else if (/^9\./.test("${{ inputs.version }}")) {
+              return process.env.BEATS_EA_FLEET_9
+            } else {
+              return ""
+            }
+          result-encoding: string
+        env:
+          EA_FLEET_8: |
+            - [ ] @${{ inputs.release_manager }} review this PR to make sure the correct items are included
+            - [ ] @elastic/ingest-docs review the language
+            - [ ] @elastic/ingest-docs open a PR in the ingest-docs repo combining the Elastic Agent and Fleet Server release notes targeting the `${{ steps.get-minor.outputs.result }}` branch
+            - [ ] @${{ inputs.release_manager }} merge on release day
```

When the content has been transferred to the ingest-docs repo. these PRs are no longer needed, and they can be merged or closed.  Maybe remove this step to avoid confusion?

---

I'm concerned it's more confusing to _not_ say anything about what to do with the PR. Maybe this?

```suggestion
            - [ ] @${{ inputs.release_manager }} merge or close on release day
                * Note: Release notes for 8.x versions are published to elastic.co/guide from the ingest-docs repo. Merging this PR in this repo won't hurt, but it also won't be published to the docs site.
```